journal/modernized/35-chapter-32.adoc

```diff
@@ -989,7 +991,7 @@ them that believe to the saving of the soul,
 notwithstanding all our profession of religion before men.
 
 This is the substance of the testimony I had to bear amongst these people,
-though but a part of what I had to say.--Inviting and encouraging them, above all things,
+though but a part of what I had to say. Inviting and encouraging them, above all things,
```

Comment:
I'm not sure this was the right thing to do . It seems as he almost meant for a pause there, but with the period before the dashes it doesn't work. Maybe it should be a semicolon and keep the dashes?